PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -200,6 +202,13 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     resolution = JME::JetResolution::get(setup, m_jets_algo_pt);
                     resolution_sf = JME::JetResolutionScaleFactor::get(setup, m_jets_algo);
                 }
+
+                if(m_useDeterministicSeed) {
+                    unsigned int runNum_uint = static_cast <unsigned int> (event.id().run());
+                    unsigned int evNum_uint = static_cast <unsigned int> (event.id().event()); 
+                    std::uint32_t seed = runNum_uint + 4*evNum_uint;
+                    m_random_generator.seed(seed);
```

Comment:
I do not see why one would want to add lumisection. In the MC example, even though we repeat Run/Event numbers, the underlying MC event is different and therefore wouldn't the affect of the random number be different?